readme.md

```diff
@@ -2,43 +2,40 @@
 
 > [Kap](https://github.com/wulkano/kap) plugin - Upload recordings to [Dropbox](https://dropbox.com)
 
-
 ## Install
 
-In the `Kap` menu, go to `Preferences…`, select the `Plugins` pane, find this plugin, and click `Install`.
-
+In the `Kap` menu, go to `Preferences…`, select the `Plugins` pane, find this plugin, and toggle it.
 
 ## Usage
 
 In the editor, after recording, select a format, and then `Upload to Dropbox`.
 
 ## Options
 
-In the `plugins` pane, click `Open plugins folder` and edit the `kap-dropbox.json` file to change options
+In the “Plugins” pane, click the pencil icon to edit the plugin options.
 
-Available options:
+Alternatively, you can manually edit the options in your text editor; Click `Open plugins folder` and edit the `kap-dropbox.json` file to change options.
```

Comment:
I just noticed that we no longer have this button, so maybe the whole section here should be removed?

Reply:
We moved it into each plugin's config window instead. I'll update 👍

Reply:
Ah, right. Duuh.